LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Not sure if we're legally required to keep it since we originally had the code here... We shouldn't need the license if we're just pulling calaccess_raw in as a dependency (though the code in `example/` is still from upstream). Maybe it does make sense to start the git repo anew.